Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM python:3.6-alpine
+FROM python:3.6-slim
```

Comment:
Does bumping to 3.8 break anything?

Reply:
The Pipfile is based on 3.6, so presumably it wouldn't be installed?

Reply:
Sorry @berroar I should have marked this as a comment not a request for change.

Reply:
Sorry @berroar I should have marked this as a comment not a request for change.